plugins/cilium-cni/cni-install.sh

```diff
@@ -6,12 +6,13 @@ HOST_PREFIX=${HOST_PREFIX:-/host}
 CNI_CONF_NAME=${CNI_CONF_NAME:-10-cilium.conf}
 MTU=${MTU:-1450}
 
-CILIUM_CNI=${CILIUM_CNI:-${HOST_PREFIX}/opt/cni/bin/cilium-cni}
+CILIUM_CNI=${CILIUM_CNI:-${HOST_PREFIX}/opt/cni/bin/}
 CILIUM_CNI_CONF=${CILIUM_CNI_CONF:-${HOST_PREFIX}/etc/cni/net.d/${CNI_CONF_NAME}}
 
-# Install cilium-cni binary tohost
+# Install cilium-cni and loopback binary to host
 echo "Installing $CILIUM_CNI ..."
 cp /opt/cni/bin/cilium-cni ${CILIUM_CNI}
+cp /opt/cni/bin/loopback ${CILIUM_CNI}
```

Comment:
We could also just install the binary release here:

    To install the CNI-loopback plugin, go to the https://github.com/containernetworking/cni/releases and get your appropiate release, e.g. for Gnu Linux/amd64:
    
    curl -fsSL 'https://github.com/containernetworking/cni/releases/download/v0.4.0/cni-amd64-v0.4.0.tgz' | tar xvz -C /opt/cni/bin/